server/bytestream/bytestream.go

```diff
@@ -53,28 +56,58 @@ func streamFromUrl(ctx context.Context, url *url.URL, grpcs bool, callback func(
 		return err
 	}
 	defer conn.Close()
-	client := bspb.NewByteStreamClient(conn)
 
-	// Request the file bytestream
-	req := &bspb.ReadRequest{
-		ResourceName: strings.TrimPrefix(url.RequestURI(), "/"), // trim leading "/"
-		ReadOffset:   0,                                         // started from the bottom now we here
-		ReadLimit:    0,                                         // no limit
-	}
-	readClient, err := client.Read(ctx, req)
-	if err != nil {
-		return err
-	}
+	if strings.Contains(url.String(), "/blobs/ac/") {
```

Comment:
Instead of doing this, you could change the url scheme above on these URLs to actioncache:// (instead of bytestream://). That way you have an unambiguous way to identify which type of content you'll be fetching

Reply:
there are a couple of places within the code where we check for the bytestream prefix, so i tweaked those as well to include the actioncache prefix but this if statement may be more succinct, lmk after you see the changes